compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1929,4 +1929,11 @@ object messages {
     val explanation =
       hl"An ${"object"} cannot have the same name as an existing ${"package"}. Rename either one of them."
   }
+
+  case class StaticFieldsOnlyAllowedInObjects(field: Symbol)(implicit ctx: Context) extends Message(StaticFieldsOnlyAllowedInObjectsID) {
```

Comment:
Substitute field by member. It is not necessarily a field. Can be a class, a val...